pyrolite/data/geochem/refcomp/PAAS_TaylorMcLennan1985.csv

```diff
@@ -0,0 +1,95 @@
+var,value,units,unc_2sigma,constraint_var,constraint_value,constraint_comment
+ModelName,PAAS_TM1985,,,,,
+Reservoir,PostArcheanAustralianShale,,,,,
+ModelType,Average,,,,,
+Reference,Taylor & McLennan 1985, McLennan 2001,,,,
```

Comment:
The reference here is split across the value and and units columns - add them to the first with a semicolon separator.